app/models/miq_server/configuration_management.rb

```diff
@@ -36,10 +37,4 @@ def config_activated(data)
 
     save
   end
-
-  def reset_server_caches
-    worker_manager.sync_config
-    sync_assigned_roles
```

Comment:
Copying over @Fryguy's concern about the ordering of this as it is something we need to confirm: https://github.com/ManageIQ/manageiq/pull/22196#discussion_r1009841832

> I'm worried that the order of this matters, because roles are stored in config (for better or worse). Might not matter though if both are synced.